pkgs/by-name/da/datalad-gooey/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  git,
+  python3,
+  fetchFromGitHub,
+  darwin,
+  stdenv,
+  git-annex,
+  qt6,
+}:
+
+python3.pkgs.buildPythonApplication {
```

Comment:
Maybe this also need to move inside python-modules (http://docs.datalad.org/projects/gooey/en/latest/modref.html). And add application using `toPythonApplication`  (https://nixos.org/manual/nixpkgs/unstable/#topythonapplication-function). I think it's also needed for `datalad`.

Reply:
done